dialog-el.html

```diff
@@ -355,7 +357,9 @@
       }.bind(this));
     }
 
-    proto.close = function() {
+    proto.close = function(e) {
+      // Allow persistent dialogs if the element clicked on has an open-dialog attribute.
+      if (e && e.target && e.target.attributes["open-dialog"]) return;
```

Comment:
Ohhh, I see the issue. This is a bug.

If you'll remove this feature, I will work on fixing the bug and we can get this merged.